### PR TITLE
Train 284 (master)

### PR DESCRIPTION
**Pull request by falseperfection**
#2177 Remove package list from cloud config

### CODE REVIEW

**Comment by falseperfection** [comment]
@mesosphere-mergebot bump-ee

**Comment by falseperfection** [comment]
@mesosphere-mergebot label ready for review

File: dcos_installer/config_util.py [hunk 1]
```diff
@@ -19,10 +19,9 @@ def make_serve_dir(gen_out):
     subprocess.check_call(['mkdir', '-p', SERVE_DIR])
     gen.build_deploy.bash.generate(gen_out, SERVE_DIR)
 
-    # Copy artifacts out of the cache.
-    generated_packages = gen_out.config_package_ids + [gen_out.arguments['custom_check_bins_package_id']]
+    # Copy cached artifacts.
     cached_packages = sorted(
-        info['filename'] for info in gen_out.cluster_packages.values() if info['id'] not in generated_packages
+        i['filename'] for i in gen_out.cluster_packages.values() if i['filename'] not in gen_out.stable_artifacts
```
Comment (orsenthil) [inline dcos_installer/config_util.py]: FWIW, `info` was better than `i`.

**Comment by orsenthil** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/docker https://jira.mesosphere.com/browse/DCOS-19542

**Comment by orsenthil** [comment]
@mesosphere-mergebot merge-it

**Pull request merged**